jcache/commons/src/main/java/org/infinispan/jcache/Closeables.java

```diff
@@ -0,0 +1,40 @@
+package org.infinispan.jcache;
+
+import org.infinispan.commons.logging.LogFactory;
+import org.infinispan.jcache.logging.Log;
+
+import java.io.Closeable;
+import java.util.Set;
+
+class Closeables {
+
+   private static final Log log = LogFactory.getLog(Closeables.class, Log.class);
+
+   Closeables() {
```

Comment:
You probably meant `private Closeables()` right? There is no need to instantiate this class. TBH, I think we could even get rid of this constructor, the class has default access after all...

Reply:
Private constructor is what I was aiming for.